flag_test.go

```diff
@@ -488,7 +488,7 @@ func TestStringFlagHelpOutput(t *testing.T) {
 		fl := &StringFlag{Name: test.name, Aliases: test.aliases, Usage: test.usage, Value: test.value}
 		// create a tmp flagset
 		tfs := flag.NewFlagSet("test", 0)
-		fl.Apply(tfs)
+		_ = fl.Apply(tfs)
```

Comment:
No need to check err return here ?

Reply:
44 instances of this in flag_test.go: 
![image](https://user-images.githubusercontent.com/42463301/226141950-cff3875d-d4a5-4008-8070-b04fcba81542.png)